projects/npm-tools/packages/npm-scripts/src/jsp/lintJSP.js

```diff
@@ -12,16 +12,18 @@ const linter = new Linter();
 
 const {major, minor} = getDXPVersion() || {};
 
-const ecmaVersion =
-	major === undefined || major > 7 || (major === 7 && minor > 3) ? 6 : 5;
+const modern = major === undefined || major > 7 || (major === 7 && minor > 3);
```

Comment:
When do you guess `modern` will become `evenMoreModern`? 😂

Reply:
I don't know but I can't wait!

![so-excited](https://user-images.githubusercontent.com/7074/107987857-14094f80-6fcf-11eb-8969-ce10b7008cba.gif)

PS: the irony of the fact that `modern` maps to ES6 is not lost on me. 😁